packages/react-components/src/TextArea.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017-2020 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import styled, { css } from 'styled-components';
+
+interface Props {
+  withError?: boolean;
+}
+
+const TextInput = css(({ withError }: Props) => `
```

Comment:
Really don't like these components al all. They are a nightmare to maintain if we want to add and enhance and will require a full re-do. Create an actual component.